### PR TITLE
Update IT translation

### PR DESCRIPTION
**Pull request by deframet**

### CODE REVIEW

File: it.po [hunk 1]
```diff
@@ -514,9 +514,10 @@ msgid ""
 "#0000ff;\">http://www.ej-technologies.com/products/jprofiler/overview.html</"
 "span></a></p></body></html>"
 msgstr ""
-"<html><head/><body><p><a href=\"http://github.com/MultiMC/MultiMC5\"><span "
```
Comment (TheDoctorsLife) [inline it.po]: This

File: it.po [hunk 2]
```diff
@@ -526,9 +527,9 @@ msgid ""
 "text-decoration: underline; color:#0000ff;\">http://visualvm.java.net/</"
 "span></a></p></body></html>"
 msgstr ""
-"<html><head/><body><p><a href=\"http://github.com/MultiMC/MultiMC5\"><span "
```
Comment (TheDoctorsLife) [inline it.po]: These too.

File: it.po [hunk 3]
```diff
@@ -538,9 +539,9 @@ msgid ""
 "decoration: underline; color:#0000ff;\">http://www.mcedit.net/</span></a></"
 "p></body></html>"
 msgstr ""
-"<html><head/><body><p><a href=\"http://github.com/MultiMC/MultiMC5\"><span "
```
Comment (TheDoctorsLife) [inline it.po]: And this.

**Comment by deframet** [comment]
@TheDoctorsLife Do you mean template.pot?

**Comment by peterix** [comment]
That file should always be in sync with the code in the stable branch. If it isn't, then there's a problem - but it should not be fixed manually, it should be fixed in the translation automation scripts.

**Comment by deframet** [comment]
So I have to translate it again in weblate?

**Comment by peterix** [comment]
I would prefer weblate, but the voting system just doesn't work with the number of active translators.

Just don't mess with the template - it's generated from code.

**Comment by peterix** [comment]
If you need to change the english text, do it in the MultiMC5 develop branch.

**Comment by deframet** [comment]
👍

**Comment by peterix** [comment]
Some source code file :)

Anyway, this didn't work:
```
build #72 of translations-build started (including [4af462acf8bfbe679213c460c07cf5defdd03d30])
build #72 of translations-build is complete: Failure [failed build translations]  Build details are at http://ci.multimc.org/builders/translations-build/builds/72  blamelist: Davide Pierotti <deframet@users.noreply.github.com>
```

And the details:
http://ci.multimc.org/builders/translations-build/builds/72/steps/build%20translations/logs/stdio

This is exactly the reason why I would prefer weblate.
If you edited the .po file by hand... please don't do that. Use some translation editor like https://poedit.net/

**Pull request merged**